python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -713,7 +726,7 @@ def _make_apdbQuery(butler, instrument, sqlitefile=None, postgres_url=None, name
     return apdb_query
 
 
-def select_sources(apdb_query, limit):
+def select_sources(apdb_query, limit, spuriousnessMin, spuriousnessMax):
```

Comment:
I suggest making these kwargs for cases when someone is using it from a notebook.

```suggestion
def select_sources(apdb_query, limit, spuriousnessMin=None, spuriousnessMax=None):
```

Reply:
Also, you need to document the new arguments in the docstring below